enterprise/server/raft/store/store.go

```diff
@@ -1753,7 +1753,11 @@ func (s *Store) SplitCluster(ctx context.Context, req *rfpb.SplitClusterRequest)
 
 	s.log.Infof("splitlog: acquired split lease")
 
-	if err := bringup.CloneCluster(ctx, s.apiClient, bootStrapInfo, req.GetRange().GetRangeId(), newIDs.rangeID); err != nil {
+	cloneReq := rbuilder.NewBatchBuilder().Add(&rfpb.CloneClusterRequest{
```

Comment:
Will this hold up properly if the CloneClusterRequest is replayed from the raft log? I don't believe Clone is idempotent.